src/libsystemd-network/sd-dhcp6-client.c

```diff
@@ -187,6 +188,33 @@ int sd_dhcp6_client_set_mac(
         return 0;
 }
 
+int sd_dhcp6_client_set_prefix_delegation_address(
+                sd_dhcp6_client *client,
+                const struct in6_addr *pd_address) {
```

Comment:
How about to merge these two setters to `sd_dhcp6_client_set_prefix_delegation_hint()`?

Reply:
yes we can do here.